staging/src/k8s.io/kubectl/pkg/explain/v2/templates/plaintext_test.go

```diff
@@ -41,17 +41,46 @@ var (
 	//go:embed apiextensions.k8s.io_v1.json
 	apiextensionsJSON string
 
+	//go:embed batch.k8s.io_v1.json
+	batchJSON string
```

Comment:
Any chance to find what you were looking for in one of the existing file? I'd rather not add new files if not strictly necessary (but let's take the file if there's no other choice).

Reply:
The difference is, that we need to test a namespaced resource and apiextensions CRDs are not namespaced. 

Also I think it is better to have a separate file for better maintenance.

```shell
$ curl localhost:6443/openapi/v3/apis/batch/v1 > batch.k8s.io_v1.json
```

This is just needed for updating the file. Merging these 2 together would be a bit harder.